skbio/io/fastq.py

```diff
@@ -449,9 +449,9 @@ def _alignment_to_fastq(obj, fh, variant=None, phred_offset=None,
 
 def _line_generator(fh):
     for line in fh:
-        line = line.rstrip('\n')
+        line = line.strip()
         if not line:
-            raise FASTQFormatError("Found blank line in FASTQ-formatted file.")
+            raise FASTQFormatError("Found blank line within FASTQ record.")
```

Comment:
Can you update this error message to indicate blank or whitespace-only (similar to FASTA error message)? It only mentioned blank before because `rstrip('\n')` was being used, but now all leading/trailing whitespace is stripped.